### DISCUSSION TITLE
sql优化工具目前默认是SQLAdvisor，有地方教程如何配置为默认为SOAR

### DISCUSSION THREAD

**Discussion by jikehaohao** [hhyo/Archery#2735] [General]
![image](https://github.com/user-attachments/assets/fdd2f336-be8f-4146-971b-73b92a5510e2)

**Comment by LeoQuote** [comment]
现在不支持, 欢迎 pr, 可以用前端逻辑保存在本地

**Reply by jikehaohao** [reply]
好的，多谢，我回头看看怎么搞！
